Source/Python/application/main.py

```diff
@@ -26,20 +31,18 @@ def resources_folder(self) -> Path:
         return self._resources_folder
 
     @property
-    def locations_request_handler(self) -> LocationsRequestHandler:
+    def locations_request_handler(self) -> LocationsRestRequestHandler:
```

Comment:
These getters should not be required anymore, as they are no longer needed by the flask app

Reply:
Removed these and the fields.